labs/src/main/scala/org/scalalabs/basic/lab02/ListManipulationExercise01.scala

```diff
@@ -15,15 +15,14 @@ object ListManipulationExercise01 {
    * 
    */
   def firstElementInList[T](l: List[T]): T = {
-    //buildin
-    null.asInstanceOf[T]
+    l.head
```

Comment:
This (and all the similarly formatted ones) can omit the dot: `l head`, `l sum`, etc. That’s probably the more Scala-y way to represent it.